src/gui/gtk.h

```diff
@@ -405,6 +405,8 @@ gboolean dt_gui_show_standalone_yes_no_dialog(const char *title, const char *mar
 char *dt_gui_show_standalone_string_dialog(const char *title, const char *markup, const char *placeholder,
                                            const char *no_text, const char *yes_text);
 
+gboolean dt_gui_show_yes_no_dialog(const char *title, const char *format, ...);
```

Comment:
/* returns TRUE if YES is selected in the dialog, FALSE otherwise */

Reply:
Maybe the name should also be more indicative of function. How about `dt_gui_confirm_with_dialog`?